spec/replication/util/miq_pglogical_spec.rb

```diff
@@ -142,4 +142,19 @@
       expect(described_class.node_name_to_region("region_5")).to eq(5)
     end
   end
+
+  describe ".save_remote_region" do
```

Comment:
These are runnable without pglogical being installed so can you put all the other specs and the `before` block for skipping them into a separate context so that these will run on travis?

Reply:
👍